### DISCUSSION TITLE
Alacritty with nushell as default shell fails to run on MacOS

### DISCUSSION THREAD

**Discussion by mock1328** [nushell/nushell#14121] [Q&A] [answered]
Sorry, I'm new to both MacOS and nushell. 
I can't run the Alacritty with `nu` defined as `shell` in `~/.config/alacritty` by clicking on Alacritty icon. It just fails silently.
If I set the shell to `zsh` OR start `alacritty` from default terminal application (with `shell = "nu"`), it starts without problems.
I've read the gettng started docs for `nushell`, and it mentions the need to specify the `XDG_CONFIG_HOME` before the run, so I tried specifying it in my `~/.zshrc`, but it didn't help

**Comment by mock1328** [comment]
I also tried setting `XDG_DATA_HOME` as proposed in #13704, just in case, in an `alacritty.toml` config but the behavior is the same.

**Comment by ghost** [comment, marked as answer]
Im not a macOS user, but here is a theory:

`nu` might not be in your global PATH (or however its called in macOS).  
Another terminal might append it to the PATH, which explains why it works when you launch alacrity from within another terminal.  

A fix if that's the case: use a absolute path instead of `nu`. you can get the absolute path using `which nu | get 0.path`

**Reply by mock1328** [reply]
Thanks. Yeah, I found that it is indeed a problem with PATHs.

Alacritty seems to inherit `PATH` from the shell it uses. But since `nushell` (installed via `brew`) is in `/opt/homebrew/bin` and it's not a standard PATH: alacritty can't find it and fails to start.

I tried adding `/opt/homebrew/bin` into `/etc/paths`, but it doesn't seem to have any effect (I don't see new variables in PATH). Dunno how to overcome this, honestly

**Reply by mock1328** [reply]
nvm, just set the absolute path as you've suggested :D
